packages/react-scripts/config/webpack.config.js

```diff
@@ -270,17 +294,34 @@ module.exports = function(webpackEnv) {
       // Automatically split vendor and commons
       // https://twitter.com/wSokra/status/969633336732905474
       // https://medium.com/webpack/webpack-4-code-splitting-chunk-graph-and-the-splitchunks-optimization-be739a861366
-      splitChunks: {
-        chunks: 'all',
-        name: false,
-      },
+      // splitChunks: {
+      //   chunks: 'all',
+      //   name: false,
+      // },
+      splitChunks: bpkReactScriptsConfig.enableAutomaticChunking
+        ? {
+            chunks: 'all',
+            name: false,
+            cacheGroups: bpkReactScriptsConfig.vendorsChunkRegex
+              ? {
+                  vendors: {
+                    test: new RegExp(bpkReactScriptsConfig.vendorsChunkRegex),
+                  },
+                }
+              : {},
+          }
+        : {},
       // Keep the runtime chunk separated to enable long term caching
       // https://twitter.com/wSokra/status/969679223278505985
       // https://github.com/facebook/create-react-app/issues/5358
       runtimeChunk: {
-        name: entrypoint => `runtime-${entrypoint.name}`,
+        // name: entrypoint => `runtime-${entrypoint.name}`,
+        name: bpkReactScriptsConfig.enableAutomaticChunking
```

Comment:
This is a breaking change for consumers - previously, consumers would have set this to `true`. The expected value is now a function which takes the entrypoint and returns a string i.e. `entrypoint => `runtime-${entrypoint.name}`.

Reply:
Ok will update the migration guide, however I don't think we ever had a default so I guess if it isn't set it should be null rather than `entrypoint => `runtime-${entrypoint.name}``?

Reply:
Actually, i have just realised...

BRS7 has:
```
// runtimeChunk: true,
runtimeChunk: bpkReactScriptsConfig.enableAutomaticChunking,
```

This is short hand for disabling the runtime chunk unless the consumer has set `enableAutomaticChunking` to `true` in their `package.json`.

BRS8 (This PR) has:
```
runtimeChunk: {
    name: entrypoint => `runtime-${entrypoint.name}`,
},
```
That means we need to have:
```
runtimeChunk: bpkReactScriptsConfig.enableAutomaticChunking 
    ? {
        name: entrypoint => `runtime-${entrypoint.name}`,
    } : false,
```